Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1361,7 +1361,8 @@ extension BrowserViewController: QRCodeViewControllerDelegate {
 
 extension BrowserViewController: SettingsDelegate {
     func settingsOpenURLInNewTab(_ url: URL) {
-        self.openURLInNewTab(url, isPrivileged: false)
+        let forcedPrivate = Preferences.Privacy.privateBrowsingOnly.value || PrivateBrowsingManager.shared.isPrivateBrowsing
```

Comment:
`PrivateBrowsingManager.shared.isPrivateBrowsing` should be good enough here, as this is set if `privateBrowsingOnly` is `true`. Were you seeing different behavior?

Reply:
You are right. Thank you. fixed. (btw copy & paste from 'NavigationRouter')